### PR TITLE
Release v8.5.0

### PR DESCRIPTION
**Pull request by fubarhouse**
This release covers the following Pull Requests, and this PR will be updated with more granular information after testing has begun.

- #122 - [GOVCMSD8-598] - Updated the CI base image
- #129 - [GOVCMSD8-594] - Added nginx security headers
- #131 - [GOVCMS-4353, GOVCMS-4200] - Add scaffold tooling to base image
- #133 - [GOVCMS-4353, GOVCMS-4200] - Update site alias for scaffold-tooling compatibility
- #134 - [GOVCMS-4353, GOVCMS-4200] - Pin scaffold-tooling
- #135  - [GOVCMSD8-576] - Added override for temporary file removal
- #136 - [GOVCMSD8-604] - Allow disabling dev mode on non-production environments 
- #139 - [GOVCMS-4245] - Added support for `COMPOSER_AUTH`
- #141 - [GOVCMSD8-594] - Typo corrections
- #142 - [GOVCMSD8-535] - PHP 7.3  & Solr 7.5 upgrade
- #143 - [GOVCMSD8-603] - Solr upgrade path
- #147 - [GOVCMSD8-603] - Permissions fix for config directory
- #150 - [GOVCMSD8-603] - hook_update for Solr upgrade

### CODE REVIEW

**Comment by tobybellwood** [comment]
we need the changes in .env to match the release

```
# Set the PHP version to use for the upstream dockerfiles (currently 7.2) - 7.3 in testing.
PHP_IMAGE_VERSION=7.3

# Set the version of the site audit script to use - you can use a tag or branch reference here
# See https://github.com/govCMS/audit-site/releases
SITE_AUDIT_VERSION=7.x-3.x

# Set the version of GovCMS and Drupal Core to use - you can use a tag or branch reference (1.x-dev) here
# Note: the DRUPAL_CORE_VERSION here must match the version required in the corresponding GOVCMS_PROJECT_VERSION
# See https://github.com/govCMS/govcms8/releases
GOVCMS_PROJECT_VERSION=release/1.2.0-dev
DRUPAL_CORE_VERSION=8.8.5

# Set the Lagoon tag to use for the upstream dockerfiles (e.g. v1.2.0)
# See https://github.com/amazeeio/lagoon/releases
LAGOON_IMAGE_VERSION=v1.4.0
```

**Comment by fubarhouse** [comment]
I'm all over it 🙂

**Comment by fubarhouse** [comment]
Waiting for a process to finish but I expect to resume issue parity remediation shortly.

File: .docker/images/govcms8/settings/production.settings.php [hunk 1]
```diff
@@ -39,3 +39,6 @@
     $config['environment_indicator.indicator']['name'] = 'Edit Domain';
   }
 }
+
+// Disable temporary file deletion (GOVCMSD8-576).
+$config['system.file']['temporary_maximum_age'] = 0;
```
Comment (Ahmedjabar) [inline .docker/images/govcms8/settings/production.settings.php:44]: Can the value be defined in an environment variable?

Comment (fubarhouse) [reply .docker/images/govcms8/settings/production.settings.php:44]: At this time no, we should raise a separate ticket if that is important.
See #135 for full details.

**Pull request merged**